### DISCUSSION TITLE
Vanilla http module wrapper

### DISCUSSION THREAD

**Discussion by Llorx** [peers/peerjs#1231] [Ideas & Feature requests]
### I have a suggestion:
Use another existant ws/HTTP server instead of express.

Express is nice, but sometimes is oversize when you just want to serve some static files and have a dumb WebSocket, so I created my own HTTP/ws server with nodejs `http` module and npm `ws` library. The problem is that I cannot share same port/server with PeerServer, as the only wrapper I can find is the express one.

Is there a way to have PeerServer working with them? If not, I suggest to add that feature.

**Comment by CICCIOSGAMINO** [comment]
I'd like to try different server too .... for example Koa instead of Express ..

**Comment by devyaz** [comment]
For me I serve Express on one port for static files and Peer on another port no problems
